releases/net471/KnownIssues/470006-GC Crashes with high rate of large object allocation.md

```diff
@@ -7,9 +7,20 @@ intermittent crashes that did not occur when running on other .NET Framework ver
 
 ## Cause
 
-.NET Framework 4.7.1 includes changes that improve background GC performance.
-Because of these changes, a rare combination of factors can cause the background GC
-to reclaim a large object while the program is still using it.
+.NET Framework 4.7.1 includes changes that improve background GC performance. It allows large object allocations during part of the BGC sweep phase and the large objects allocated during this time must be marked correctly in the allocator or it will be erroneously reclaimed. A GC can be triggered during the large object allocator code path and if the following happens this bug will be triggered:
+
+- this is a BGC 
+
+- the range for BGC changed from the previous range (ie, a new segment is allocated outside of the previous range)
```

Comment:
the --> The (each list item should begin with a capital letter)
end in period here and for each list item